airflow/decorators/__init__.pyi

```diff
@@ -327,7 +328,7 @@ class TaskDecoratorCollection:
         api_version: str | None = None,
         container_name: str | None = None,
         cpus: float = 1.0,
-        docker_url: str = "unix://var/run/docker.sock",
+        docker_url: str = environ.get("DOCKER_HOST") or "unix://var/run/docker.sock",
```

Comment:
I don't think we should add expressions into the stub file. It is not evaluated in runtime.